pkg/ingestor/collector/transparency/transparency.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2022 The AFF Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transparency
+
+const (
+	CollectorTransparency = "Transparency"
```

Comment:
i think this would be named specific to rekor? since implementations of different tlogs would be different enough - like we'd have one for rekor and scitt for example? thoughts?

Reply:
Makes sense. I was thinking they would be similar to this would capture rekor and others that come in the future. If that is not going to be the case I can change to be just rekor.